src/components/Guilds/ProposalPage/ProposalInfoCard/index.tsx

```diff
@@ -1,5 +1,9 @@
+import moment, { unix } from 'moment';
+import React, { useMemo } from 'react';
 import { FiCheck, FiInbox } from 'react-icons/fi';
+import Skeleton from 'react-loading-skeleton';
 import styled from 'styled-components';
+import { useProposal } from '../../../../hooks/Guilds/proposals/useProposal';

```

Comment:
@mprasanjith This hook is not the one using EtherSWR. 
that one is located at hooks/Guilds/useProposal
I just let it there because we may not be ready to move away from Guilds/proposals/useProposal